.editorconfig

```diff
@@ -0,0 +1,4 @@
+root = true
+
+[*.py]
+multi_line_output=5
```

Comment:
ruff has an [Isort plugin](https://beta.ruff.rs/docs/rules/#isort-i), so we should either remove this or ignore the `I` ruleset.

Reply:
Removed this setting. Since `.editorconfig` is now empty, also removed the file to keep things simple.